tests/data/test344

```diff
@@ -45,14 +45,13 @@ http://%HOSTIP:%HTTPPORT/344 --etag-compare log/etag344 --etag-save log/etag344
 ^User-Agent:.*
 </strip>
 <protocol>
-GET /344 HTTP/1.1
-Host: %HOSTIP:%HTTPPORT
-Accept: */*
-If-None-Match: ""
-
+GET /344 HTTP/1.1
+Host: %HOSTIP:%HTTPPORT
+Accept: */*
+
```

Comment:
I think you removed the CRLF line endings here? The test 344 (and 345) failures look like it:https://github.com/curl/curl/pull/5700/checks?check_run_id=990398257#step:7:1446

Reply:
damn, probably this was accidentally changed. Can you fix it? I'm too busy

Reply:
ok, I manually switched the file to CRLF